sdk/test/NetStandard/IntegrationTests/IntegrationTests/Route53Domains.cs

```diff
@@ -45,11 +45,11 @@ public async Task TestTagging()
             var domains = (await Client.ListDomainsAsync()).Domains;
             if (domains.Count > 0)
             {
-                await TestTagging(Client, domains[0].DomainName);
+                await TestTaggingAsync(Client, domains[0].DomainName);
             }
         }
 
-        private async Task TestTagging(AmazonRoute53DomainsClient client, string domain)
+        private async Task TestTaggingAsync(AmazonRoute53DomainsClient client, string domain)
```

Comment:
I renamed this with the `Async` because for some reason the new version of XUnit didn't like that there was a test method and a private method with the same name in the same class.